core/src/javascript/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import RegistrationComponent from './registrations';
+
+var mountElement = document.createElement('div');
+ReactDOM.render(<RegistrationComponent/>, mountElement);
```

Comment:
Why do you need to render a component here ? Is it only to get access to i18next ? In this case it should be possible to directly call i18next.t()

Reply:
Also, you can wrap this code in a callback that will be called when the appshell is ready - this has been added recently for content editor : 
```
registry.add('callback', 'external-provider', {
    targets: ['jahiaApp-init:50'],
    callback: () => {
        registerExternalProviderRoutes();
        registerMountpointsRoutes()
    }
}
```
This allows to control the order of loading of you registration (with target priority), which can be useful if you have dependencies between them

Reply:
Yes, for i18n mainly. I'll try the callback.

Reply:
The only thing with directly calling i18next.t() is that it may not be ready yet (although init() method should have been called already, by jahia-ui-root). Don't know how it behaves, but hopefully it should work